site/docs/master/faq.md

```diff
@@ -0,0 +1,11 @@
+# Frequently Asked Questions
+
+## Conformance testing
+### Why were so many tests skipped?
+<answer>
+
+### Why did my test fail?
+<answer>
+
+### How can I configure settings when running the E2E tests?
```

Comment:
I really like how you're first just figuring out the questions instead of spending more time on the answers right away. I'll try and suggest some in the PR too